hudi-utilities/pom.xml

```diff
@@ -139,14 +125,22 @@
         </exclusion>
       </exclusions>
     </dependency>
+
+    <!-- Hoodie - Other -->
+
+    <!-- NOTE: This dependency is kept around even though it's not necessary, due to the fact
+               that removing it messes up Orc dependencies on the class-path failing to compile
+               test for Spark 3.x (this occurs b/c Hudi depends on "nohive" version of "orc-core"
+               while Spark 3.x switches to a regular one) -->
     <dependency>
       <groupId>org.apache.hudi</groupId>
-      <artifactId>hudi-spark-common_${scala.binary.version}</artifactId>
+      <artifactId>hudi-common</artifactId>
```

Comment:
hudi-spark pulls in hudi-common as well right. do we need to explicitly depend here?

Reply:
@nsivabalan the note above explained it